Assignment1/tests/game.test.ts

```diff
@@ -0,0 +1,24 @@
+import { expect, test } from 'vitest'
+import { Game } from '../scripts/game'
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(1 + 2).toBe(3);
+})
+
+test('check string for name', () => {
+  expect('Hello, World!').toMatch('Hello');
+})
+
+test('check object for property', () => {
+  expect({ name: 'John' }).toHaveProperty('name');
+})
+
+test('check game guess word property', () => {
+  const game = new Game('hello');
+  expect(game.wordToGuess).toBe('hello');
+})
```

Comment:
```suggestion
test('game initializes wordToGuess property as expected ', () => {
  const game = new Game('hello');
  expect(game.wordToGuess).toBe('hello');
})
```
Taking the opportunity here to note that it's valuable to get in the habit of naming your tests so that they describe what is being tested as well as possible.